tests/slow/src/test/scala/tests/gradle/GradleLspSuite.scala

```diff
@@ -284,7 +284,7 @@ object GradleLspSuite extends BaseImportSuite("gradle-import") {
       _ = assertNoDiff(
         client.messageRequests.peekLast(),
         // only 3 projects since no empty test targets are created for gradle
```

Comment:
> only 3

now it's 5 ;)

Reply:
Updated the comment not to include a number - that was not important really.